CMakeLists.txt

```diff
@@ -1,21 +1,30 @@
-set(_ct_min_cmake_version "3.14" CACHE INTERNAL "")
-cmake_minimum_required(VERSION "${_ct_min_cmake_version}") #Required for FetchContent_MakeAvailable()
+#3.19 needed because of CMaize
+set(_ct_min_cmake_version "3.19" CACHE INTERNAL "")
+cmake_minimum_required(VERSION "${_ct_min_cmake_version}")
 project(CMakeTest VERSION 1.0.0 LANGUAGES NONE)
 
 if(NOT CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
-	set(_ct_min_cmake_version "${_ct_min_cmake_version}" PARENT_SCOPE) #Make sure projects using CMakeTest have this variable
+    #Make sure projects using CMakeTest have this variable
+	set(_ct_min_cmake_version "${_ct_min_cmake_version}" PARENT_SCOPE)
 endif()
 
+# Add cmake directory to CMake's module path, iff it's not already there
```

Comment:
Is the `iff` here a typo, or intended to mean "if and only if"?

Reply:
I meant it as the latter.